.travis.yml

```diff
@@ -1,4 +1,3 @@
 sudo: false
-install: git clone --depth 1 https://github.com/sstephenson/bats.git
-script: PATH="./bats/bin:$PATH" test/run
```

Comment:
What's weird, is that as soon as I include `test_helper` the `helper_test`, it fails in development:

```
✗ helpers search aliases
   (in test file test/lib/helpers.bats, line 15)
     `[[ "${lines[0]}" =~ 'plugins: base' ]]' failed
   cat: /var/folders/dz/r8zs2sh953qd1k19gr0q3tj00000gn/T/.bash_it/plugins/available/*.bash: No such file or directory
```

Maybe this is a clue?  Why wouldn't it find anything in that folder?